packages/workflow-transformer-apply-arguments-to-fields/src/index.js

```diff
@@ -10,13 +10,14 @@ function parseValue(value, args) {
 }
 
 export default class WorkflowTransformerApplyArgumentsToFields {
-  async transformBefore() {}
   async transformAfter(node, { args }) {
     switch (node.type) {
       case 'app': {
         const { open, ...rest } = node;
         return { open, ...mapValues(rest, v => parseValue(v, args)) };
       }
+      default:
+        return node;
```

Comment:
That sound like the start of a good idea, it might be a valid use-case to want to delete subtree though. I've made #142 for discussing the issue out of this PR.